### DISCUSSION TITLE
Advertising on Github - what are the policies?

### DISCUSSION THREAD

**Discussion by danpolanco** [community/community#23045] [New to GitHub]
<p>Today, I ran into <a>awesome-ios</a>. I immediately noticed that it has an advertisement on it. It’s easy on the eyes in my opinion.</p>
<p>However, I wanted to know if there is a policy for such things? If not, is this something the community should address?</p>
<p>It doesn’t seem out of hand now. But it could get quite annoying if it spreads.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/e59c97c7725493196c8f99629efe3f9a811a7a02.png" data-download-href="/uploads/short-url/wLeQ2Pv5t2F1mgbCqnen6ii2O1I.png?dl=1" title="Screen Shot 2018-08-17 at 12.56.29.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3826143/181074522-7675a029-6bc5-4a3a-bfd8-1130e0cd2b10.png" alt="Screen Shot 2018-08-17 at 12.56.29.png" data-base62-sha1="wLeQ2Pv5t2F1mgbCqnen6ii2O1I" width="540" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/e59c97c7725493196c8f99629efe3f9a811a7a02_2_540x500.png, https://aws1.discourse-cdn.com/github/optimized/1X/e59c97c7725493196c8f99629efe3f9a811a7a02_2_810x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/e59c97c7725493196c8f99629efe3f9a811a7a02_2_1080x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/e59c97c7725493196c8f99629efe3f9a811a7a02_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-08-17 at 12.56.29.png</span><span class="informations">2464×2280 706 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by angela-crist** [comment]
<p>Hi <a class="mention" href="/danthecoloradan">@danthecoloradan</a>,</p>
<p>This is a great question. I am going to move this post to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/danthecoloradan">@danthecoloradan</a>,</p>
<p>I’d recommend taking a look at GitHub’s Terms of Service, specifically the <a href="https://help.github.com/articles/github-terms-of-service/#k-advertising-on-github" rel="nofollow noopener">section on advertising</a>. The tl;dr of that is "<em>We do not generally prohibit use of GitHub for advertising. However, we expect our users to follow certain limitations, so GitHub does not become a spam haven."</em></p>
<p>I hope this helps!</p>

**Reply by smbproton** [reply]
@nadiajoyce I think that the section on advertising you linked has been moved or removed. I'd be happy to learn more about advertising on GitHub.

**Comment by cck42** [comment]
<p>Did you ever find anything out about this? Interested as well.</p>

**Comment by rachelwalker78** [comment]
<p>I would like to enquire about advertising on GitHub. We are recruiting on behalf of a Tech client for approx. 100 roles.</p>
<p>The roles will be mainly software engineers (of all levels), full stack engineers, tech delivery gameplay programmers, game designers, artists and tech delivery managers so your audience is ideal, so wondered what advertising / sponsorship or partnership opportunities there may be?  Many thanks.</p>

**Comment by webbethany93** [comment]
Advertising should be discreet and not intrusive. Github users expect to see quality code here, not ads. You can not use mass mailing, spam, create false profiles or fake repositories for advertising. I read advertisement analysis before, used [https://phdessay.com/free-essays-on/advertisement-analysis/]( https://phdessay.com/free-essays-on/advertisement-analysis/) for this. Developers can try using Github to promote their brand, such as creating a repository for their project, blogging about the development, or getting their company noticed through participation in Github events and conferences. I learned this when I myself learned how everything works here. It turns out that not everything is as simple as it might seem.

**Comment by the-moog** [comment]
What about a repository that contains no code has only one or two commits and they are just exits to a README.md file. It appears be for no other purpose than to pick up search results. e.g. https://github.com/ApotheoTech/CXP_ENC

**Comment by SoeAung95** [comment]
Pls fix

**Comment by SoeAung95** [comment]
Pls fix
